charts/i2b2-wildfly/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "i2b2-wildfly.fullname" . }}
+  labels:
+    {{- include "i2b2-wildfly.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "i2b2-wildfly.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "i2b2-wildfly.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "i2b2-wildfly.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.http }}
+              protocol: TCP
+            - name: api
+              containerPort: {{ .Values.service.api }}
+              protocol: TCP
+          livenessProbe:
+            {{- toYaml .Values.livenessProbe | nindent 12 }}
+          readinessProbe:
+            {{- toYaml .Values.readinessProbe | nindent 12 }}
+          env:
+          {{- range $name, $value := .Values.env }}
+            {{- if typeIs "map" $value }}
+            - name: {{ $name }} 
+              valueFrom: 
+                secretKeyRef: 
+                  name: {{ $value.existingSecret }} 
+                  key: {{ $value.key }} {{- else }}
+            - name: {{ $name }} 
+              value: "{{ $value }}" 
+            {{- end }} 
+          {{- end }}
```

Comment:
```suggestion
          env:
            {{- range $key, $value := .Values.env }}
            {{- if kindIs "map" $value }}
            {{- if and (hasKey $value "existingSecret") (hasKey $value "key") }}
              - name: {{ $key }}
                valueFrom:
                  secretKeyRef:
                    name: {{ $value.existingSecret }}
                    key: {{ $value.key }}
            {{- end }}
            {{- else }}
              - name: {{ $key }}
                value: "{{ $value }}"
            {{- end }}
            {{- end }}
```

Reply:
This should help

Reply:
Error: couldn't find key ds_wb_pass in Secret i2b2/i2b2-wildfly-secret